forge-ai/src/main/java/forge/ai/ability/RepeatAi.java

```diff
@@ -54,16 +52,16 @@ protected boolean doTriggerAINoCost(Player ai, SpellAbility sa, boolean mandator
         if (sa.usesTargeting()) {
             if (logic.startsWith("CopyBestCreature")) {
                 Card best = null;
-                if (!ai.getGame().getStaticEffects().getGlobalRuleChange(GlobalRuleChange.noLegendRule) || logic.endsWith("IgnoreLegendary")) {
-                    best = ComputerUtilCard.getBestAI(Iterables.filter(ai.getCreaturesInPlay(), Predicates.and(
-                            CardPredicates.isTargetableBy(sa), new Predicate<Card>() {
-                                @Override
-                                public boolean apply(Card card) {
-                                    return !card.getType().isLegendary();
-                                }
-                            })));
+                Iterable<Card> targetableAi = Iterables.filter(ai.getCreaturesInPlay(), CardPredicates.isTargetableBy(sa));
+                if (logic.endsWith("IgnoreLegendary")) {
```

Comment:
Looks like this AILogic was wrong before
"IgnoreLegendary" is used when the copy is not, so it should be inverted